### DISCUSSION TITLE
[MAUI] TabHostView: Tabs are still selectable when IsEnabled="False" on iOS

### DISCUSSION THREAD

**Discussion by timjamwoo** [roubachof/Sharpnado.Tabs#135] [General]
**Platform (please complete the following information):**
 - OS: iOS
 - Device: iPhone 13 Pro Max
 - Sdk version: iOS 17.2.1
 - MAUI: 8.0.6

**Describe the bug**
When using TabHostView, BottomTabItems are still selectable when IsEnabled="False". They correctly appear disabled, but when tapping the "disabled" items, we are still brought to that tab. The issue doesn't affect Android, or TabButtons.

**Comment by roubachof** [comment]
Use the IsSelectable property instead of the IsEnabled  property
